src/main/java/org/folio/circulation/domain/MoveRequestService.java

```diff
@@ -72,7 +86,14 @@ private RequestAndRelatedRecords pagedRequestIfDestinationItemAvailable(
 
   private CompletableFuture<Result<RequestAndRelatedRecords>> updateRelatedObjects(
       RequestAndRelatedRecords requestAndRelatedRecords) {
+
     return updateUponRequest.updateItem.onRequestCreateOrUpdate(requestAndRelatedRecords)
       .thenComposeAsync(r -> r.after(updateUponRequest.updateLoan::onRequestCreateOrUpdate));
   }
+
+  private Function<Result<RequestAndRelatedRecords>, CompletionStage<Result<RequestAndRelatedRecords>>> getTimeZoneForRequestRelatedRecords() {
```

Comment:
This method seems to be the same as `CreateRequestService#getTimeZoneForRequestRelatedRecords`. Can we move it to `ConfigurationRepository`, probably creating a method similar to `ConfigurationRepository#lookupTimeZoneForLoanRelatedRecords`, but for `RequestAndRelatedRecords`?

Reply:
It is indeed the same. After receiving [this feedback](https://github.com/folio-org/mod-circulation/pull/338#discussion_r321250739) on the previous pull request, I chose to prefer a little duplication over introducing more coupling into the configuration repository.

Since then, I have introduced an overload of the Result.combineAfter method to make these a little simpler, and removed the references to the related records classes from the configuration repository.

Personally I prefer this direction as I think the repositories have become too embroiled in the process mechanics.

Do you think this introduces too much duplication? If so, I suggest that we extra process adaptor methods to accommodate that. Would that work for you?

Eventually, I'd like to remove the related records classes, they were a tactical means to an end that I think are turning into an anti-pattern as the codebase grows.

What do you think?